scripts-available/CDB_ForeignTable.sql

```diff
@@ -0,0 +1,122 @@
+---------------------------
+-- FDW MANAGEMENT FUNCTIONS
+--
+-- All the FDW settings are read from the `cdb_conf.fdws` entry json file.
+---------------------------
+
+CREATE OR REPLACE FUNCTION cartodb._CDB_Create_FDW(name text, config json)
```

Comment:
minor thing: there are a few var/params called `name` in this file. I'd rename this one to be called `srvname` or similar for clarity's sake